libcloud/compute/drivers/runabove.py

```diff
@@ -86,11 +78,41 @@ def list_nodes(self, location=None):
         return self._to_nodes(response.object)
 
     def ex_get_node(self, node_id):
+        """
+        Get a individual node.
+
+        :keyword node_id: Node's ID
+        :type    node_id: ``str``
+
+        :returns: Created node
+        :rtype  : :class:`Node`
+        """
         action = API_ROOT + '/instance/' + node_id
         response = self.connection.request(action, method='GET')
         return self._to_node(response.object)
 
     def create_node(self, **kwargs):
```

Comment:
Sorry, I just mean you should declare all the arguments in the method signature, e.g. `def create_node(self, name, image, size, location, ex_keyname=None)`.